packages/reactivity/__tests__/ref.spec.ts

```diff
@@ -80,6 +85,24 @@ describe('reactivity/ref', () => {
     expect(typeof (c.value.b + 1)).toBe('number')
   })
 
+  it('should pass type check on ref', () => {
+    const a: AssertEqual<UnwrapRef<[number, Ref<string>, Ref<Ref<number>>]>,
```

Comment:
I think there's an easier way to do it 😅 

``` js
const arrRef = ref([1, ref(1)])
let [a, b] = arrRef
a++
b++
```

Reply:
We are testing the type, not the real value, right?

Reply:
Yes. As long as the test compiles it means the types are correct.

Reply:
Okey, I get that